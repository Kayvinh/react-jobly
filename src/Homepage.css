.Homepage{
    text-shadow: 2px 2px 5px #34302F;
    display: flex;
    height: 90vh;
    justify-content: center;
    align-items: center;
}