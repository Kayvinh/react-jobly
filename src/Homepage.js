import React from 'react'

const Homepage = () => {
  return (
    <div className='Homepage'>
        <div className="container text-center">
            <h1 className='mb-4 fw-bold'>Jobly</h1>
            <p className='lead'> All the jobs in one, convenient place.</p>
        </div>
    </div>
  )
}

export default Homepage