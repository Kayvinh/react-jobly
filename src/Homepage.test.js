import React from 'react';
import { render } from '@testing-library/react';
import Homepage from './Homepage';

test('renders without crashing', function () {
    const { container } = render(
      <Homepage />
    );
});

test('renders proper text', function () {
    const { container } = render(
      <Homepage />
    );

    expect(container).toContainHTML("Jobly")
    expect(container).toContainHTML("All the jobs in one, convenient place.")
});

test('matches snapshot', function() {
    const {container} = render(
        <Homepage />
    );

    expect(container).toMatchSnapshot()

});