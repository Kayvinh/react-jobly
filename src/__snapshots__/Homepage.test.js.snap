// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Homepage d-flex justify-content-start"
  >
    <div
      class="container text-center"
    >
      <h1
        class="mb-4 fw-bold"
      >
        Jobly
      </h1>
      <p
        class="lead"
      >
         All the jobs in one, convenient place.
      </p>
    </div>
  </div>
</div>
`;
